cloud/logger/integration/index.ts

```diff
@@ -0,0 +1,12 @@
+
+import {BunyanLogger, Logger} from '../src/index';
+
+// constructor accept bunyan options. Please refer to https://www.npmjs.com/package/bunyan
+const log: Logger = new BunyanLogger({name: 'index'});
+// by default debug logging is turned off to enable debug logging change constructor to
```

Comment:
...turned `off. In order to` enable debug logging, change...
Should maybe add this comment above the Logger declaration.